Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM golang:1.11.6-alpine3.9
```

Comment:
Bummer, go hasn't matured enough to have LTS versions yet. Was hopeful 1.11 or 1.12 was one so we could lock this in on an LTS.